src/features/controls.ts

```diff
@@ -302,6 +303,11 @@ export const ControlsMixin = <T extends Constructor<ModelViewerElementBase>>(
       }
 
       this[$controls].update(time, delta);
+      const target = this.getCameraTarget();
+      if (!this[$scene].pivotCenter.equals(target)) {
+        this[$scene].pivotCenter.copy(target);
+        this[$scene].setRotation();
```

Comment:
Here we fall back to the argument default, which is tricky to understand without reading the method signature. But, it's equivalent to what we are doing on line 280 because the default we just added is the same value. Let's just drop the default and specify the value here instead.